makefile.cargo

```diff
@@ -10,7 +10,7 @@ CONFIGURE_FLAGS := \
 
     CONFIGURE_FLAGS += \
         --with-default-fonts=/system/fonts \
-        --with-sysroot=$(ANDROID_TOOLCHAIN)/sysroot \
+        --with-sysroot=$(ANDROID_NDK)/platforms/android-18/arch-arm/ \
```

Comment:
It might be better to set an `$ANDROID_SYSROOT` variable in Servo's mach commands, based on the `android.platform` and `android.target` options from the `servobuild` file, rather than hard-coding `android-18` and `arch-arm`.